internal/usage/usage.go

```diff
@@ -107,6 +107,7 @@ dbName and collection are required only for built-in roles.`
 	MaxDate                                   = "Maximum created date. This option returns events whose created date is less than or equal to the specified value."
 	MinDate                                   = "Minimum created date. This option returns events whose created date is greater than or equal to the specified value."
 	ClusterFilename                           = "Path to an optional JSON configuration file that defines cluster settings. To learn more about cluster configuration files for the Atlas CLI, see https://dochub.mongodb.org/core/cluster-config-file-atlascli. To learn more about cluster configuration files for MongoCLI, see https://dochub.mongodb.org/core/mms-cluster-settings-file-mcli."
+	IndexFilename                             = "Path to an optional JSON configuration file that defines index settings."
```

Comment:
do you intend to have documentation for the file structure? if so make sure to ask for a dochub link and add it here

Reply:
I think we need the doc team to create the doc page and then we can update the description here. Let me know if I am missing something

Reply:
you can have the link ahead of time if you ask them and once the page is up they redirect it. we don't this way before

Reply:
You can alternatively open a docsp ticket and ask them to update this line, make sure to add a link to where they should add the link if that is the approach you prefer.

Reply:
Unfortunately, we cannot use the example in the open api spec (it does not work) and I haven't found any examples in the doc. The doc team will need to use the files in the data folders included in this PR 😭